_posts/2023-07-07-kubernetes-tokens-openfaas-api.md

```diff
@@ -102,23 +105,17 @@ We have thorough documentation on the REST API endpoints, a Go SDK and an OpenAP
 
 If your code is running in a Kubernetes cluster, then you can get the initial `id_token` from a service account. You'll need to create a dedicated Service Account and then bind it to the Pods that need to access the OpenFaaS API.
 
+Before tokens issued by the Kubernetes API can be exchanged for OpenFaaS tokens the Kubernetes API will have to be registered as a trusted JWT issuer with OpenFaaS. The OpenFaaS chart should have created the JwtIssuer object on installation.
 
-Before tokens issued by the Kubernetes API can be exchanged for OpenFaaS tokens the Kubernetes API will have to be registered as a trusted JWT issuer with OpenFaaS:
+You can list issuers and look for the kubernetes issuer to verify it is registerd with OpenFaaS:
 
-```yaml
```

Comment:
I'd still like this to be shown and detailed in the article, even if it is outputted by the Helm chart. This is part of the story.

Reply:
I updated the PR and added instructions to show the issuer via kubectl